### PR TITLE
Documented the values and secrets for the notification and SMTP services

### PR DESCRIPTION
**Pull request by lnowotny**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/install-systemlink-enterprise/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Documents the secrets and values need to configure the notification and SMTP services

### Why should this Pull Request be merged?

This documentation is needed for the next GM deployment

### What testing has been done?

N/A

### CODE REVIEW

File: getting-started/templates/systemlink-secrets.yaml [hunk 1]
```diff
@@ -280,6 +280,32 @@ nbexecservice:
...
 
+## Secret configuration for notifications
+##
+notification:
+  secrets:
+    mongodb:
+      ## Root user password for the database cluster.
+      ##
+      rootPassword: "" # <ATTENTION>
+      ## Password for the limited user allocated to allow the service to access the database. This password cannot contain commas or any character which must be escaped in a URL.
```
Comment (MelissaHilliard) [inline getting-started/templates/systemlink-secrets.yaml:291]:
```suggestion
      ## Limited user password to allow the service to access the database. This password cannot contain commas or any character that must be escaped in a URL.
```

File: getting-started/templates/systemlink-values.yaml [hunk 2]
```diff
@@ -699,3 +699,26 @@ license:
...
+    ## Optional - The SMTP server host.
+    ##
+    # host: "example"
+    ## Optional - The SMTP server port.
+    ##
+    # port: 1025
+    ## Optional - The from address used in notification emails.
+    ##
+    # fromAddress: "noreply@example.com"
+    ## Optional - Whether SSL should be enabled or not.
```
Comment (MelissaHilliard) [inline getting-started/templates/systemlink-values.yaml:715]:
```suggestion
    ## Optional - Whether SSL should be enabled.
```

File: getting-started/templates/systemlink-values.yaml [hunk 3]
```diff
@@ -699,3 +699,26 @@ license:
...
+    ##
+    # enableSsl: true
+    ## Optional - Whether credentials are required to connect to the SMTP server. If this is true, the smtp-server-credentials secret must exist.
+    ##
+    # requireAuthentication: true
+    ## Optional - The timeout in seconds when communicating with the SMTP server.
```
Comment (MelissaHilliard) [inline getting-started/templates/systemlink-values.yaml:721]:
```suggestion
    ## Optional - The timeout, in seconds, when communicating with the SMTP server.
```

File: getting-started/templates/systemlink-secrets.yaml [hunk 4]
```diff
@@ -280,6 +280,32 @@ nbexecservice:
...
+      ## Refer to mongodb documentation for key generation: https://www.mongodb.com/docs/manual/tutorial/enforce-keyfile-access-control-in-existing-replica-set/#create-a-keyfile
+      ##
+      replicaSetKey: "" # <ATTENTION>
+
+## Secret configuration for emails
+##
+smtp:
+  secrets:
+    smtpServer:
+      ## Optional - Credentials used to access the SMTP server
```
Comment (JD-Robertson) [inline getting-started/templates/systemlink-secrets.yaml:304]: Maybe explain here that this can be left unconfigured if SMTP is not enabled?

**Review by JD-Robertson — APPROVED** [review]

**Pull request merged**